pom.xml

```diff
@@ -95,6 +95,7 @@
     <cassandra.driver.version>3.6.0</cassandra.driver.version>
     <kms.version>1.20.0</kms.version>
     <proto-kms.version>0.67.0</proto-kms.version>
+    <parquet.version>2.14.0</parquet.version>
```

Comment:
This should be removed. Please use ${beam.version} for the beam-sdks-java-io-parquet depdendency.